nimterop.nimble

```diff
@@ -59,20 +59,24 @@ when (NimMajor, NimMinor, NimPatch) >= (0, 19, 9):
     fmt"{currentSourcePath}".parentDir.parentDir.parentDir
 
 proc runNimDoc() =
-  execCmd &"nim doc -o:{htmldocsDir} --project --index:on nimterop/all.nim"
-  execCmd &"nim buildIndex -o:{htmldocsDir}/theindex.html {htmldocsDir}"
-  when declared(getNimRootDir):
-    #[
-    this enables doc search, works at least locally with:
-    cd {htmldocsDir} && python -m SimpleHTTPServer 9009
-    ]#
-    execCmd &"nim js -o:{htmldocsDir}/dochack.js {getNimRootDir()}/tools/dochack/dochack.nim"
+  when not defined(windows): # pending https://github.com/nim-lang/Nim/pull/10629
+    execCmd &"nim doc -o:{htmldocsDir} --project --index:on nimterop/modules.nim"
+    execCmd &"nim buildIndex -o:{htmldocsDir}/theindex.html {htmldocsDir}"
+    when declared(getNimRootDir):
+      #[
+      this enables doc search, works at least locally with:
+      cd {htmldocsDir} && python -m SimpleHTTPServer 9009
+      ]#
+      execCmd &"nim js -o:{htmldocsDir}/dochack.js {getNimRootDir()}/tools/dochack/dochack.nim"
 
 task test, "Test":
   buildToast()
   testAll()
   runNimDoc()
 
+task nimDoc, "run nim doc":
+  runNimDoc()
+
```

Comment:
Why a second task?

Reply:
for users (eg me) who want to run docs locally but don't have access to your github pages account

Reply:
o, didn't realize i had already added `task docs, "Generate docs":` ; will remove

Reply:
done